base/src/org/compiere/util/EMail.java

```diff
@@ -625,7 +628,6 @@ private void setEMailParameters(int eMailConfigId) {
 		}
 		//	Validate Config
 		if(eMailConfigId == 0) {
-			MClient client = MClient.get(ctx);

```

Comment:
Here get the Client from the wrong context (System).
Now the Email Server Configuration is getted from the correct Client